packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -469,6 +475,17 @@ export default decorate([
               </div>
             </fieldset>
           )}
+          <div className='form-group'>
+            <label>{_('remoteEncryptionKey')}</label>
```

Comment:
We should add an *alpha* tag for this feature and a description (either a small paragraph or a tooltip) to explain the limit of this feature.

Reply:
- all the files of the remote except the encryption.json are encrypted, that means you can only activate encryption or change key on an empty one
- you won't be able to get your data back if you lose the encryption key
- the encryption key is saved in the XO config backup, they should be secured correctly. Be carefull, if you saved it on an encrypted remote, then you won't be able to access it without the remote encryption key.
- delta backup must use Vhd saved as blocks (note  should be enforced when saving settings)
- size of backup is not updated when using encryption